### ISSUE TITLE
a11y: Quantity change causes auto submit

### ISSUE THREAD

**Issue by khinkelthein-dotsource** [shopware/shopware#19871]
### Technical TODO

When changing the quantity in offcanvas cart, cart and checkout, the form auto submits to update the cart data. This violates the following WCAG criteria:

- 3.2.2 On Input: 
  - https://www.w3.org/TR/WCAG22/#on-input
  - https://www.w3.org/WAI/WCAG22/Techniques/failures/F36
- 3.2.5 Change on Request: 
  - https://www.w3.org/TR/WCAG22/#change-on-request
  - https://www.w3.org/WAI/WCAG22/Understanding/change-on-request

To avoid that:

- a visible submit button (like "update quantity") should be included
- the form should be submittable via "Enter" key, when focus is on the quantity input field

**Steps to reproduce**

- add product to cart
- open the offcanvas cart
  - change the quantity either by using the buttons or direct input
- navigate to the cart page
  - change the quantity either by using the buttons or direct input
- navigate to checkout confirm page
  - change the quantity either by using the buttons or direct input (this has to be enabled)

**Comment by mstegmeyer** [comment]
We have deemed that the auto-submit itself is ok, also as focus is restored. In #20055, we still implement some improvements regarding input behavior and debouncing.
